src/cli/utils/tests/async-load-config.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { unlinkSync, writeFileSync } from 'fs';
+import { asyncLoadConfig } from '../async-load-config';
+import {
+	asyncDataWithAsyncFunctionAndAsyncParameter,
+	asyncFunctionWithAsyncData,
+	asyncyFunctionWithNonAsyncData,
+} from '../fixtures/asyn-load-config-test-fixture';
+
+describe('async load tests', () => {
+	it('should aysnc load medusa-config with non-async', async () => {
+		const data = asyncyFunctionWithNonAsyncData;
+
+		const pathToConfigFile = `${process.cwd()}/../medusa-config.js`;
+		writeFileSync(pathToConfigFile, data);
+		const configModule = await asyncLoadConfig();
+		expect(configModule).toBeDefined();
+		expect(configModule.projectConfig).toBeDefined;
```

Comment:
you must check that each props have the expected value, otherwise we can't be sure it works properly just with the test

Reply:
same for the other tests